src/cacheout/cache.py

```diff
@@ -442,6 +442,29 @@ def expire_times(self) -> t.Dict[t.Hashable, T_TTL]:
         """
         with self._lock:
             return self._expire_times.copy()
+        
+    def item_ttl(self, key: t.Hashable) -> T_TTL:
```

Comment:
I think `get_ttl` would be good for the method name. If expanding on this kind of API, then could align with other methods like `set_ttl` and `has_ttl`.